scripts/release.sh

```diff
@@ -0,0 +1,281 @@
+#!/bin/bash
+
+# George Miller
+# 05-17-2022
+# If you don't have permission to run, try: chmod u+x Release.sh
+# numbering follows this document: https://github.com/smartdevicelink/sdl_ios/wiki/Release-Steps
+# The numbering does restart halfway because the document does.
+
+# script start
+echo 
+#echo "Zug Zug"
+echo "Starting SDL release script..."
+
+# If we are running from the scripts directory, we want to pop back to the project root to do everything.
+if [[ $PWD == *"scripts" ]]; then
+    cd ..
+fi
+# If, for soem reason, we are not now in the correct working directory, exit
+if [[ $PWD != *"sdl_ios" ]]; then
+    echo "Please run this from the sdl_ios project root or the sdl_ios/scripts directory"
+    exit 0
+fi
+
+
+# 1 bump version in projectFile
+echo
+echo "Step 1: Update version in Project File"
+
+# 1.1) get the current version and build from the podspec file
+project_file=./SmartDeviceLink-iOS.xcodeproj/project.pbxproj
+new_file=./SmartDeviceLink-iOS.xcodeproj/new.pbxproj
+current_version_number=$(sed -n '/MARKETING_VERSION/{s/MARKETING_VERSION = //;s/;//;s/^[[:space:]]*//;p;q;}' $project_file)
+current_build_number=$(sed -n '/CURRENT_PROJECT_VERSION/{s/CURRENT_PROJECT_VERSION = //;s/;//;s/^[[:space:]]*//;p;q;}' $project_file)
+echo "Current Version "$current_version_number
+echo "Current Build "$current_build_number
```

Comment:
```suggestion
echo "Current Version: "$current_version_number
echo "Current Build: "$current_build_number
```

These were missed from the last review.